src/System.Net.Requests/src/System/Net/FtpWebRequest.cs

```diff
@@ -1576,7 +1576,7 @@ public bool KeepAlive
         {
             get
             {
-                return false;
```

Comment:
It is more of an implementation detail in `FtpWebRequest` with the control channel.

Comment from Desktop code:

>If KeepAlive is set to false, then the control connection to the server will be closed when the request completes